### PR TITLE
Refacto/few improvments

### PR DESCRIPTION
**Pull request by tanguyfalconnet**
## Why
Linter is not happy.
Insecure ssh connection are dirty.

## How
Add possibility to add a know_hosts file to ssh secure connection.
Handle linter reviews
Use logger instead of fmt

## Breaking changes
No

New release tag : 1.0.1

### CODE REVIEW

**Review by vgallissot — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Usage of netapp-exporter:
   interval: 30                                  # Collect Interval
   username: "XXX"                               # Netapp SSH Username
   password: "XXX"                               # Netapp SSH Password
+  known_hosts_file: "/root/.ssh/known_hosts"    # For SSH Secure Connection
```
Comment (taufort) [inline README.md]: Why are u using root to connect through SSH? Cannot u use a non-root user?

File: collector/collector.go [hunk 2]
```diff
@@ -51,13 +53,21 @@ func RecordMetrics(username string, password string, netappH string, netappP int
...
 	sshConfig := &ssh.ClientConfig{
 		User:    username,
 		Auth:    []ssh.AuthMethod{ssh.Password(password)},
 		Timeout: time.Second * 30,
 	}
 	sshConfig.HostKeyCallback = ssh.InsecureIgnoreHostKey()
+	if knownHostsFile != "" {
+		callback, err := knownhosts.New(knownHostsFile)
+		if err != nil {
+			fmt.Println(err)
```
Comment (taufort) [inline collector/collector.go]: You should use a logger instead of fmt :slightly_smiling_face:

File: README.md (hunk 1 above)
Comment (tanguyfalconnet) [reply README.md]: This is only for example :) Any path with known_hosts is good

File: collector/collector.go (hunk 2 above)
Comment (tanguyfalconnet) [reply collector/collector.go]: Done ✅

**Pull request merged**